lib/krane/kubernetes_resource/deployment.rb

```diff
@@ -91,6 +91,10 @@ def deploy_timed_out?
       return false if deploy_failed?
       return super if timeout_override
 
+      if progress_condition.present?
+        StatsD.client.increment('fail_fast', 1, tags: statsd_tags)
```

Comment:
might be a good idea to keep the `kubectl` prefix, so it's clear that this metric is specific to kubectl and it's easier to discover